### PR TITLE
add error message when running from a non-existent directory

### PR DESCRIPTION
**Pull request by tp-woven**
# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

File: cli/src/cli_util.rs [hunk 1]
```diff
@@ -2664,7 +2664,10 @@ impl CliRunner {
         ui: &mut Ui,
         mut layered_configs: LayeredConfigs,
     ) -> Result<(), CommandError> {
-        let cwd = env::current_dir().unwrap(); // TODO: maybe map_err to CommandError?
+        let cwd = env::current_dir().expect(
+            "Could not determine current directory. Did you check-out a commit where the \
+             directory doesn't exist?",
+        ); // TODO: maybe map_err to CommandError?
```
Comment (tp-woven) [inline cli/src/cli_util.rs:2670]: Could also map to a CommandError, if that's better?

Comment (yuja) [reply cli/src/cli_util.rs:2670]: Yes, `user_error()` should be indeed better.

Comment (tp-woven) [reply cli/src/cli_util.rs]: Done.

**Review by yuja — APPROVED** [review]
Thanks

**Review by ilyagr — APPROVED** [review]
Thank you very much!

**Pull request merged**